CMakeLists.txt

```diff
@@ -372,7 +372,7 @@ if(CMAKE_USE_DARWINSSL)
   message(FATAL_ERROR "The cmake option CMAKE_USE_DARWINSSL was renamed to CMAKE_USE_SECTRANSP.")
 endif()
 
-if(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
+if(${CMAKE_SYSTEM_NAME} MATCHES "Darwin" OR ${CMAKE_SYSTEM_NAME} MATCHES "iOS")
```

Comment:
So how about ipadOS, tvOS, watchOS etc?

Reply:
Maybe switch back to pre 62be096085 status? Then the check looked like this:
~~~
if(CMAKE_USE_SECTRANSP)

Reply:
Added tvOS and watchOS (not tested though). ipadOS is not a known CMAKE_SYSTEM_NAME

Reply:
So the CoreFoundation and SystemConfiguration things are needed on those platforms no matter which TLS backend is used?

Reply:
We only have CoreFoundation calls in the Secure Transport code AFAIK.

Reply:
@rasmartins you don't seem to have acknowledged this?

Reply:
@bagder I apologize for the silence, I got sidetracked and forgot about this issue. Changed the PR to address the comments